config/settings.yml

```diff
@@ -1095,6 +1095,16 @@ lighthouse:
     url: https://sandbox-api.va.gov
     path: services/benefits-reference-data
     version: v1
+  direct_deposit:
+    host: https://sandbox-api.va.gov
+    access_token:
+      aud_claim_url: https://deptva-eval.okta.com/oauth2/ausi0e21hbv5iElEh2p7/v1/token
+      api_scopes:
```

Comment:
I would avoid putting anything that doesn't change across environments here. The main reason being that you will have to put these into the AWS parameter store so they can be used in the EKS configs